NPE in

Reported on JetBrains marketplace: https://plugins.jetbrains.com/plugin/13234-quarkus-tools/reviews#review=75851

```
Plugin version: 1.14.0.292 shows the following errors.

ExternalToolPass: Details: root_path.txt D:/projects/quarkus/todo/src/main/java/rest/Todos.java

com.intellij.diagnostic.PluginException: annotator: com.redhat.devtools.intellij.quarkus.lsp4ij.operations.codeactions.LSPCodeActionAnnotator@66b203f1 (class com.redhat.devtools.intellij.quarkus.lsp4ij.operations.codeactions.LSPCodeActionAnnotator) [Plugin: com.redhat.devtools.intellij.quarkus] at com.intellij.diagnostic.PluginProblemReporterImpl.createPluginExceptionByClass(PluginProblemReporterImpl.java:23) at com.intellij.diagnostic.PluginException.createByClass(PluginException.java:83) at com.intellij.codeInsight.daemon.impl.ExternalToolPass.processError(ExternalToolPass.java:263) at com.intellij.codeInsight.daemon.impl.ExternalToolPass.collectInformationWithProgress(ExternalToolPass.java:138) at com.intellij.codeInsight.daemon.impl.ProgressableTextEditorHighlightingPass.doCollectInformation(ProgressableTextEditorHighlightingPass.java:97) at com.intellij.codeHighlighting.TextEditorHighlightingPass.collectInformation(TextEditorHighlightingPass.java:57) at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$1(PassExecutorService.java:405) at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1086) at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$2(PassExecutorService.java:397) at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeProcessUnderProgress$13(CoreProgressManager.java:591) at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:666) at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:622) at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:590) at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:60) at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.doRun(PassExecutorService.java:396) at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$run$0(PassExecutorService.java:372) at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:174) at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:182) at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.run(PassExecutorService.java:370) at com.intellij.concurrency.JobLauncherImpl$VoidForkJoinTask$1.exec(JobLauncherImpl.java:181) at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) Caused by: java.lang.NullPointerException: Cannot invoke "com.intellij.util.messages.MessageBusConnection.disconnect()" because "this.messageBusConnection" is null at com.redhat.devtools.intellij.quarkus.lsp4ij.LanguageServerWrapper.stop(LanguageServerWrapper.java:426) at com.redhat.devtools.intellij.quarkus.lsp4ij.LanguageServerWrapper.start(LanguageServerWrapper.java:342) at com.redhat.devtools.intellij.quarkus.lsp4ij.LanguageServerWrapper.getInitializedServer(LanguageServerWrapper.java:654) at com.redhat.devtools.intellij.quarkus.lsp4ij.LanguageServerWrapper.getServerCapabilities(LanguageServerWrapper.java:687) at com.redhat.devtools.intellij.quarkus.lsp4ij.LanguageServiceAccessor.lambda$getMatchingStartedWrappers$10(LanguageServiceAccessor.java:479) at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:178) at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:179) at java.base/java.util.HashMap$KeySpliterator.forEachRemaining(HashMap.java:1707) at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509) at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499) at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:921) at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:682) at com.redhat.devtools.intellij.quarkus.lsp4ij.LanguageServiceAccessor.getMatchingStartedWrappers(LanguageServiceAccessor.java:480) at com.redhat.devtools.intellij.quarkus.lsp4ij.LanguageServiceAccessor.getLSWrappers(LanguageServiceAccessor.java:301) at com.redhat.devtools.intellij.quarkus.lsp4ij.operations.codeactions.LSPCodeActionAnnotator.doCollectInformation(LSPCodeActionAnnotator.java:96) at com.redhat.devtools.intellij.quarkus.lsp4ij.operations.codeactions.LSPCodeActionAnnotator.collectInformation(LSPCodeActionAnnotator.java:82) at com.redhat.devtools.intellij.quarkus.lsp4ij.operations.codeactions.LSPCodeActionAnnotator.collectInformation(LSPCodeActionAnnotator.java:57) at com.intellij.codeInsight.daemon.impl.ExternalToolPass.collectInformationWithProgress(ExternalToolPass.java:135) ... 21 more


```

--- a/src/main/java/com/redhat/devtools/intellij/quarkus/lsp4ij/LanguageServerWrapper.java
+++ b/src/main/java/com/redhat/devtools/intellij/quarkus/lsp4ij/LanguageServerWrapper.java
@@ -423,7 +423,9 @@ public class LanguageServerWrapper {
         this.languageServer = null;
 
         EditorFactory.getInstance().getEventMulticaster().removeDocumentListener(fileBufferListener);
-        messageBusConnection.disconnect();
+        if (messageBusConnection != null) {
+            messageBusConnection.disconnect();
+        }
     }
 
     /**

